Web/DpAssignment1/designPattern1qn1/Logger.java

```diff
@@ -0,0 +1,48 @@
+package designPattern1qn1;
+
+import java.io.IOException;
+import java.util.logging.FileHandler;
+import java.util.logging.SimpleFormatter;
+
+public class Logger {
+
+	
+	private java.util.logging.Logger jLogger= java.util.logging.Logger.getLogger("MyLog");
+	private static Logger logger= new Logger();
+
+private FileHandler fh;
+private Logger()
+{
+	try {  
+	       
+        // This block configure the logger with handler and formatter  
+        fh = new FileHandler("D:/MyLogFile.log");  
```

Comment:
Strings like this (configuration etc.) should always be put in some constant (static final variable)  and then be used.